contracts/keep3r-liquidity-manager/Keep3rLiquidityManagerUserLiquidityHandler.sol

```diff
@@ -77,9 +122,14 @@ abstract contract Keep3rLiquidityManagerUserLiquidityHandler is Keep3rLiquidityM
     address _lp,
     uint256 _amount
   ) internal {
+    // TODO better revert messages
+    require(liquidityMinAmount[_lp] != 0, 'liquidity-min-not-set');
+    require(liquidityMinAmount[_lp] <= _amount, 'amount-not-enough');
```

Comment:
Is it ok to have a min amount PER user ? OR should we check if desiredAmount[_amount].add(_amount) must be > than liquidityMinAmount.

Reply:
Makes 100% sense. Since this way you can deposit and then assign tiny amounts to job, and we can get DDoSed
I'll change it